README.md

```diff
@@ -1,6 +1,8 @@
 # GOV.UK Registers Ruby Client
 
-You can use this Ruby client to integrate your service with GOV.UK Registers. 
+You can use this Ruby client to integrate your service with [GOV.UK Registers](https://registers.cloudapps.digital/).
+
+Registers are lists of information. Each register is the most reliable list of its kind. For example, the Foreign and Commonwealth Office’s (FCO’s) [country register](https://country.register.gov.uk/) is the most accurate and up-to-date list of countries available.
```

Comment:
Perhaps the claim about reliability should be constrained to the UK (potentially UK Government). After all the lists we curate are authoritative under this restriction.

Reply:
I don't disagree but I copied this from: https://registers.cloudapps.digital/

Reply:
I could easily create some new content for this but I thought it made more sense to be consistent with that content

Reply:
🤔

Reply:
Perhaps we need to revisit the original bit first 👍

Reply:
Perhaps we should consult with @danmitchell-